### ISSUE TITLE
simplejson-3.16.1-cp27-cp27m-manylinux1_x86_64.whl is not a supported wheel on this platform

### ISSUE THREAD

**Issue by isstabb** [simplejson/simplejson#237]
pip install simplejson==3.16.1 fails on CentOS 6.8.

```
bash-4.1# uname -or
2.6.32-696.6.3.el6.x86_64 GNU/Linux
bash-4.1# rpm --query centos-release
centos-release-6-8.el6.centos.12.3.x86_64
```

```
simplejson-3.16.1-cp27-cp27m-manylinux1_x86_64.whl is not a supported wheel on this platform.
Exception information:
Traceback (most recent call last):
  File "/opt/venv/lib/python2.7/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/opt/venv/lib/python2.7/site-packages/pip/commands/install.py", line 312, in run
    wheel_cache
  File "/opt/venv/lib/python2.7/site-packages/pip/basecommand.py", line 297, in populate_requirement_set
    requirement_set.add_requirement(req)
  File "/opt/venv/lib/python2.7/site-packages/pip/req/req_set.py", line 245, in add_requirement
    wheel.filename
InstallationError: simplejson-3.16.1-cp27-cp27m-manylinux1_x86_64.whl is not a supported wheel on this platform.
/
```

Seems probably related to https://github.com/simplejson/simplejson/pull/231

**Comment by etrepum** [comment]
I've deleted the 3.16.1 release for now. Other than documentation, additional wheels is the only change.

**Comment by etrepum** [comment]
Going to close this and re-open #181

**Comment by edmorley** [comment]
> I've deleted the 3.16.1 release for now.

Unfortunately we'd already merged a pyup bot PR to update to the new release, so deleting the release broke our builds. Would it be possible in the future to push a new version with the changes, rather than delete a release once published? :-)

**Comment by etrepum** [comment]
That’s generally what happens, but in this case the changes are the wheels and this was a whole lot quicker for me to do.

**Comment by fake-name** [comment]
@edmorley - 

> Unfortunately we'd already merged a pyup bot PR 

Really, pyup definitely needs to handle this case. See https://github.com/pyupio/pyup/issues/308

Unfortunately, the pyup developers seem unresponsive.

**Comment by edmorley** [comment]
> Really, pyup definitely needs to handle this case

I agree it would be handy if pyup spotted that the package was unpublished, and opened a PR to roll out back. However that's orthogonal - after all, someone not using pyup could have manually updated to the new release, and would have been in the same situation.

IMO not unpublishing packages falls into the same category as "don't force push to a widely used project's `master` branch" - it's something that should only be be done if there is no other choice. 

Anyway that said, as a maintainer of open source projects, I know it's often a thankless task and hard to always make the time to do the right thing - so no worries about this occasion, and thank you for your work on this project :-)

**Comment by fake-name** [comment]
I'd be happy if they simply stopped attempting to generate PRs for releases that don't exist. If a version has been unpublished, and you close the pyup PR, pyup will then generate another PR, ad infinitum. 

I have several projects where I have pyup PRs sitting that point to invalid releases. 

I don't disagree that it's better to not have unpublished packages, but across the ecosystem, it's going to happen, and Pyup.io not responding to issue reports for more then *two months* isn't exactly encouraging.

**Comment by isstabb** [comment]
I opened this issue and I believe it might be a red herring.  We had an environmental issue that caused the wrong unicode support for our wheels to be built.  As it happened, this was the first manylinux platform wheel that we built and hit this issue.  I've since noticed the same issue with numpy and found the root cause above, resolved the wrong unicode wheel being built and it passed our build.

Since 3.16.1 is gone I cannot retest for you, but I have a hunch it is the same red herring.
